packages/adminrouter/extra/src/test-harness/tests/test_cache.py

```diff
@@ -835,17 +832,26 @@ def assert_filter_regexp_for_invalid_app(
             filter_regexp (dict): Filter definition where key is the message
                 looked up in logs and value is SearchCriteria definition
             app (dict): App that upstream endpoint should respond with
-            nginx_class (Nginx): NGINX fixture
+            nginx_class (Nginx): SCHEDULER fixture
```

Comment:
I haven't read enough to comment but it seems like the name and the description don't match whether @mhrabovcin is right or not.